src/sandstorm/fuse.h

```diff
@@ -17,13 +17,89 @@
 #ifndef SANDSTORM_FUSE_H_
 #define SANDSTORM_FUSE_H_
 
-#include <sandstorm/fuse.capnp.h>
 #include <kj/time.h>
 #include <kj/io.h>
 #include <kj/function.h>
+#include <kj/refcount.h>
 
 namespace kj { class UnixEventPort; }
 
+namespace fuse {
+class File;
+class Directory;
+
+class Node {
+public:
+  virtual kj::Own<Node> addRef() = 0;
+
+  struct LookupResults {
+    kj::Own<Node> node;
+    uint64_t ttl;
+  };
+
+  virtual kj::Maybe<LookupResults> lookup(kj::StringPtr name) = 0;
+
+  enum class Type {
+    UNKNOWN = 0,
+    BLOCK_DEVICE = 1,
+    CHARACTER_DEVICE = 2,
+    DIRECTORY = 3,
+    FIFO = 4,
+    SYMLINK = 5,
+    REGULAR = 6,
+    SOCKET = 7,
+  };
+
+  struct Attributes {
```

Comment:
Would it make any sense to use `struct fuse_attr` directly here?

I suppose it's probably not worth the refactoring effort, but curious what you think.